specs/step-definitions/markdown-support-en-pirate.steps.ts

```diff
@@ -0,0 +1,20 @@
+import { loadFeature, defineFeature, DefineStepFunction } from '../../src';
+
+const feature = loadFeature('./specs/features/markdown-support.en-pirate.feature.md');
+
+// FIXME - code generation is not working....
+// FIXME - tests are not captured...
+// Root Cause - https://github.com/cucumber/common/blob/b43e36f5e0123023a71e5be859aef73209e2e796/gherkin/javascript/src/GherkinInMarkdownTokenMatcher.ts#L84
+// Language has to be set as a global option
```

Comment:
See: https://github.com/cucumber/common/blob/b43e36f5e0123023a71e5be859aef73209e2e796/gherkin/javascript/src/GherkinInMarkdownTokenMatcher.ts#L84

Setting the dialect explicitly when constructing the `GherkinInMarkdownTokenMatcher` I can choose which markdown dialect test fails.  The same applies for the code generation.

Questions:
* Does multi-language support need to be supported for this PR? (Can it be a future enhancement?)
* What's the preferred approach mechanism to set the dialect?
  * Adding dialect as a global option seems a sensible approach:
    * For Markdown users, if they are are using non-English dialect this is their only option.
    * Setting a global default dialect should set it for `GherkinClassicTokenMatcher` as well
    * It would require a test case (or review the gherkin test cases) to ensure that setting a default dialect in `GherkinClassicTokenMatcher` can be overridden in a classic feature file definition.
    * Setting a global default would mean that users are limited to a single dialect - a project cannot have multi-lingual Markdown.  If it was - this should be a feature request on the Gherkin project.